modules/mongodb/mongodb_test.go

```diff
@@ -13,7 +17,48 @@ import (
 	"github.com/testcontainers/testcontainers-go/modules/mongodb"
 )
 
+func getLocalNonLoopbackIP() (string, error) {
```

Comment:
suggestion: remove get prefix as it's idiomatic in go to not use get / set prefix unless required.

Reply:
Updated.